### DISCUSSION TITLE
Matrix layout in Overview

### DISCUSSION THREAD

**Discussion by KevinBowen0** [mzur/gnome-shell-wsmatrix#281] [Q&A] [answered]
Hello, I'm on Fedora 40 (Gnome 46), and when in overview mode, the workspace thumbnail display at the top of the screen correctly shows them in a gird layout, but the animation when actually switching workspaces still displays them linearly. Is this expected behavior, or is it a bug/user error?

**Comment by mzur** [comment, marked as answer]
This is a known issue since GNOME Shell 40 (https://github.com/mzur/gnome-shell-wsmatrix/issues/175). I'd be happy if someone would have a look at this.
